tests/cdc/run.sh

```diff
@@ -18,6 +18,9 @@ function prepare() {
 
     # record tso before we create tables to skip the system table DDLs
     start_ts=$(run_cdc_cli tso query --pd=http://$UP_PD_HOST_1:$UP_PD_PORT_1)
```

Comment:
Maybe we need to extract it into a function? It looks like it repeats many times. Maybe we can just awk it after it finishes running in run_cdc_cli?

Reply:
ok，good idea.